pkg/backup/item_backupper.go

```diff
@@ -302,6 +302,23 @@ func (ib *defaultItemBackupper) executeActions(
 			continue
 		}
 
+		// Determine whether to run actions for cluster-scoped resources based on IncludeClusterResources
+		// NOTE: we specifically allow actions to be run for namespaces even if IncludeClusterResources is
+		// false.
+		if namespace == "" && groupResource != kuberesource.Namespaces {
```

Comment:
I think we can simplify this whole block (L308-321) a lot - some of the checks here aren't relevant. I *believe* what we want is:

```go

if namespace == "" && !action.namespaceIncludesExcludes.IncludeEverything() {
    log.Debug("Skipping action because resource is cluster-scoped and action only applies to specific namespaces")
    continue
}
```

Reply:
Ah I see, so IncludeClusterResources is only relevant for backups and not for actions?

Reply:
Correct. It specifies whether to back up all cluster-scoped resources or not, but shouldn't have any impact on whether specific backup item actions are run for those resources or not.

Reply:
Thanks that makes sense!